example-apps/react-js/src/components/OnboardingTour.jsx

```
@@ -0,0 +1,317 @@
+        },
+      }
+    ],
+    onDestroyed: () => {
+      localStorage.setItem(TOUR_KEY, "true");
+    },
+  });
+
+  return driverObj;
+};
+
+export function TourButton({label, className}) {
```

ESLint

[error] 'className' is defined but never used.

(no-unused-vars)